app/consumers/purl_updates_consumer.rb

```diff
@@ -11,6 +11,8 @@ def process(message)
     purl = Purl.find_by!(druid: cocina_object.externalIdentifier)
     PurlCocinaUpdater.new(purl, cocina_object).update
 
+    write_public_cocina(cocina_object, purl.purl_druid_path)
+
     test_public_xml_generation(cocina_object)
```

Comment:
I don't think there's any point in continuing to test. I feel I may have been remiss in failing to point this out in the issue.  This is now "Swap testing for actually writing the file"